src/prommis/precipitate/tests/test_precipitator.py

```diff
@@ -89,10 +89,12 @@ def prec(self):
         m.fs.unit.aqueous_inlet.conc_mass_comp[0, "Cl"].fix(1e-9)
         m.fs.unit.aqueous_inlet.conc_mass_comp[0, "SO4"].fix(1e-9)
         m.fs.unit.aqueous_inlet.conc_mass_comp[0, "HSO4"].fix(1e-9)
-        m.fs.unit.aqueous_inlet.conc_mass_comp[0, "H2O"].fix(1000000)
+        m.fs.unit.aqueous_inlet.conc_mass_comp[0, "H2O"].fix(100000)
```

Comment:
Was this change intentional?

Reply:
Yes, I was creating a huge scaling issue with this example by having such a large flow